common/lib/dependabot/clients/azure.rb

```diff
@@ -206,6 +208,32 @@ def post(url, json)
 
         response
       end
+      
+      # Get pr description length in UTF-16 encoding.
+      def get_pr_description_length(pr_description)
```

Comment:
Maybe we can get the `bytesize` of the string instead, it's 4 chars for an emoji instead of 2, but I'm not super worried about truncating a bit earlier, and it'll simplify the code a lot.

Alternatively we could `pr_description.force_encoding(Encoding::UTF_16).size` and pr_description.force_encoding(Encoding::UTF_16).first(truncate_length).enforce_encoding(Encoding::UTF_8)`

Reply:
Yes i think this would simplify the code! Thanks for the suggestion